src/domain/child/ChildUtils.ts

```diff
@@ -46,8 +46,5 @@ export const getTranslatedRelationshipOptions = (
  * TODO: Fix reducer default data to match backend typing
  */
 export const getSupportedChildData = (child: Child) => {
-  const omited = omit(child, 'homeCity');
```

Comment:
This is a speling error by the way, should be "omitted"

Reply:
could be yea pls fix

Reply:
I can't correct a spell of ing error in a line of code that gets deleted :-)